tests/e2e/test_install_sh/dist/protostar/info/pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[tool.poetry]
```

Comment:
would it make sense to use a symlink here instead of coping the file?

Reply:
Actually, this file can be removed.

Reply:
Removed